pkgs/test_core/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
-## 0.5.10-wip
+## 0.6.0-wip
 
 * Handle paths with leading `/` when spawning test isolates.
+* Added `dart2wasm` as a supported compiler for the `chrome` runtime.
```

Comment:
And not firefox?

Reply:
Does it work? If it works, I can add it.

Reply:
with stable FF, yep. Haven't tried it with test yet...

Reply:
I will just add it and see what happens :)

Reply:
aaand I hit a race condition with the auto submit bot so its landed, 🤣

Reply:
but, it works!